src/phel/test.phel

```diff
@@ -10,23 +10,28 @@
 # ------
 
 (def- stats @{:failed []
-              :counts @{:failed 0
-                        :error 0
-                        :pass 0
-                        :total 0}})
+              :counts {:failed 0
+                       :error 0
+                       :pass 0
+                       :total 0}})
+
+(defn- print-report-state [state]
+  (print (case state
+           :failed "F"
+           :pass "."
+           :error "E")))
+
+(defn- should-report-println [total-columns]
```

Comment:
Rename to `report-newline?`. Functions that return a boolean should end with `?`